### ISSUE TITLE
echo only supports ports 52000-52010 / maximum of 11 devices

### ISSUE THREAD

**Issue by bparees** [toddmedema/echo#10]
I've tried adding more than 11 devices, using ports above and below 52000-52010 and this seems to totally break the echo device discovery....  it never requests the setup.xml for the devices on ports outside of 52000-52010, and I think it also prevents it from discovering the devices on the ports in range too (though they are discovered if the out of range device is deleted).

I don't think it's an issue w/ this library, but it's something to be aware of (and i'd love some confirmation that others are seeing this limitation).

**Comment by toddmedema** [comment]
Hey @bparees - interesting find! We don't have 11+ Echo devices to confirm this, but this is a good note on the maximum number of supported Echoes for future users - thank you :)

**Comment by bparees** [comment]
> We don't have 11+ Echo devices to confirm this

Right, well i was more just hoping to see if anyone else could confirm they can't create more than 11 fauxmo devices.  e.g. here's my triggers array:

```
 TRIGGERS = {
                "living room fan": 52000,
                "living room table lamps": 52001,
                "living room light": 52002, 
                "kitchen light": 52003,
                "dining room light": 52004,
                "bedroom fan": 52005,
                "bedroom light": 52006,
                "bedroom sun lamp": 52007,
                "bedroom table lamp": 52008,
                "bedroom amp": 52009,
                "bedtime": 52010,
                #"bedroom blinds": 51999,
#                "hallway blinds": 52012,
#                "kitchen blinds": 52013,
#                "living room blinds": 52014
                }
```

If i add any of the commented out entries, that's when things go wonky.  Certainly doesn't seem likely i'm doing something wrong, but who knows.

I assume if i were to put them on distinct IPs things would work, but that's more than i want to deal with.

**Comment by Monarch73** [comment]
@bparees 

In my project I'm having a bunch of guys with similar problems. All these problems seem to come from Echo plus and 2nd Gen Echos. What Echo Version do you have?

**Comment by bparees** [comment]
2nd gen echo dots.
